src/hydrusvideodeduplicator/dedup.py

```diff
@@ -372,13 +372,17 @@ def clear_trashed_files_from_db(self) -> None:
             CHUNK_SIZE = 32
             delete_count = 0
             with SqliteDict(str(DEDUP_DATABASE_FILE), tablename="videos", flag="c") as hashdb:
-                for batched_keys in self.batched(hashdb, CHUNK_SIZE):
-                    is_trashed_result = self.is_files_trashed_hydrus(batched_keys)
-                    for result in is_trashed_result.items():
-                        if result[1] is True:
-                            del hashdb[result[0]]
-                            delete_count += 1
-                    hashdb.commit()
+                total = len(hashdb)
+                with tqdm(dynamic_ncols=True, total=total, desc="Checking for trashed files in cache",
+                          unit="files", colour="BLUE") as pbar:
```

Comment:
Change unit="files" to unit="video" singular for consistency with the other tqdms

Reply:
Good call 👍🏼